package.json

```diff
@@ -1,5 +1,7 @@
 {
   "dependencies": {
+    "@casl/ability": "^6.5.0",
+    "@casl/react": "^3.1.0",
```

Comment:
@Mohith234 you have added the packages to root package.json . the y should be installed inside the ui

Reply:
+1